src/utils/graphql/mutations/createApplication.mutation.ts

```diff
@@ -1,16 +1,17 @@
 import { gql } from '@apollo/client'
 
 export default gql`
-  mutation createApplication($name: String!, $serial: String!, $templateId: Int!) {
+  mutation createApplication($name: String!, $serial: String!, $templateId: Int!,
+  $outcome: ApplicationOutcome = PENDING, $trigger: Trigger = ON_APPLICATION_CREATE ) {
```

Comment:
Are these default values? That's cool, good to know we can do this.

Reply:
Yep!